pipeline/tasks/protoc_tasks.py

```diff
@@ -540,8 +541,17 @@ def execute(self, src_proto_path, api_name,
             self.exec_command(['mkdir', '-p', output_path])
             self.exec_command([
                 'cp', '-rf', proto_path, output_path])
-        # Clean up non proto files
+
+        # Copy compiled proto files
```

Comment:
For java, does this mean ".java" files or ".class" files? The "compiled" here feels ambiguous. Is it correct to say "Copy files generated from protos"?